interfaces/builtin/basedeclaration.go

```diff
@@ -190,6 +193,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  classic-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
We could have 'deny-connection: true' but we've used that only for non-implicit interfaces historically. In this case, the interface is implicit and the base declaration says it may only be supplied by a core snap, so what is currently implemented was deemed sufficient.